RedditCrap/app/js/popup.js

```diff
@@ -52,6 +52,19 @@ function displaySavedSettings() {
 
         var selectElement = document.getElementById('crappyAction');
         selectElement.selectedIndex = data.crappyAction;
+
+    var radios = document.forms["radio-form"].elements["colour"];
+    var button;
+    for(var i = 0, max = radios.length; i < max; i++) 
+    {
+        if (radios[i].value == data.crappyColour)
+        {
+            radios[i].checked = true;
```

Comment:
We can add a `break;` behind this considering there should be only 1 colour selected at a time and save some precious clock cycles :-)